ingestion/src/metadata/ingestion/source/mysql.py

```diff
@@ -36,3 +41,24 @@ def create(cls, config_dict, metadata_config_dict, ctx):
         config = MySQLConfig.parse_obj(config_dict)
         metadata_config = MetadataServerConfig.parse_obj(metadata_config_dict)
         return cls(config, metadata_config, ctx)
+
+    def next_record(self) -> Iterable[Entity]:
+        inspector = inspect(self.engine)
+        schema_names = (
```

Comment:
@ulixius9 we should move this to prepare method instead of next_record. We only need to call this once right?